parallel-install/README.md

```diff
@@ -47,3 +47,52 @@ Once you have a configured `Deployment` instance, use the following functions ac
+Required parameters:
+| Parameter | Type     | Example value                                     | Description                                                                            |
+| --------- | -------- | ------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| src       | `string` | `myWorkspace/files/prometheus-config-updater.zip` | Path to the zip file.                                                                  |
+| dst       | `string` | `myWorkspace/files/prometheusConfigUpdater`       | Path to the output directory in which files/folders within the zip file will be moved. |
+
+`archive.Untar` decompress a `tar.gz`/`tgz` file, moving all the files and folders within the `tar.gz`/`tgz` file to an output directory.  
+Required parameters:
+| Parameter | Type     | Example value                                     | Description                                                                                       |
+| --------- | -------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| src       | `string` | `myWorkspace/files/avs-bridge-noparent-1.3.5.tgz` | Path to the `tar.gz`/`tgz` file.                                                                  |
+| dst       | `string` | `myWorkspace/files/avsBridgeNoparent`             | Path to the output directory in which files/folders within the `tar.gz`/`tgz` file will be moved. |
+
+### Git Package
+`git` package allows you to clone git repositories.  
+
+`git.CloneRepo` clones a git repository.  
+Required parameters:
+| Parameter | Type     | Example value                          | Description                                                                                                                                                             |
+| --------- | -------- | -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| url       | `string` | `https://github.com/kyma-project/kyma` | URL to the git repository.                                                                                                                                              |
+| dstPath   | `string` | `myWorkspace/repos/kyma`               | Path to which the repository will be cloned.                                                                                                                            |
+| rev       | `string` | `master`                               | Revision which is used for checking out the repository. It can be `master`, a release version (e.g. `1.4.1`), a commit hash (e.g. `34edf09a`) or a PR (e.g. `PR-9486`). |
```

Comment:
```suggestion
| rev       | `string` | `master`                               | Revision which is used for checking out the repository. It can be `master`, a release version (e.g. `1.4.1`), a commit hash (e.g. `34edf09a`), or a PR (e.g. `PR-9486`). |
```